dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -529,25 +529,25 @@ private void addValidationsToLambdaEvent(LambdaEvent event, Collection<WorkflowA
             if (isNotEmpty(validations)) {
                 String subMessage = computeMultiValidationMessage(validations);
                 if (StringUtils.isNotEmpty(subMessage)) {
-                    return String.format("In version '%s' of %s '%s': %s", workflowAndVersion.getVersionName(), workflowAndVersion.getWorkflowType().getTerm(), computeWorkflowName(workflowAndVersion), subMessage);
+                    return String.format("In version '%s' of %s '%s':\n%s", workflowAndVersion.getVersionName(), workflowAndVersion.getWorkflowType().getTerm(), computeWorkflowName(workflowAndVersion), subMessage);
```

Comment:
Looking at your screenshot in dockstore/dockstore-ui2#1524, I suggest making the first string `"In %s '%s':\n%s"` and getting rid of `workflowAndVersion.getVersionName()` as a parameter -- the version is (sort of, in the reference column) of already there, and if you had multiple workflows with validation errors for the same repository, you would keep seeing the same text again and again, e.g.,

```
In version foo of workflow one:
A validation error
In version foo of workflow two:
Another validation error
In version bar of workflow three:
A validation error
```

Reply:
I'm torn here, between your valid concern about the repeating text, and the utility of the version being included in the error message.  What's everyone else think?

Reply:
Alternatively, you could mention the version only once before all of the workflow validation messages. Something along the lines of:
```
There are errors in version foo.
In workflow one:
- A validation error
In workflow two:
- Another validation error
```
Re: Charles comment in the dockstore-ui PR about indenting the children, I propose dashes :slightly_smiling_face:

Reply:
+1 for dashes and single printout of versions.